modules/phase_field/include/kernels/MatGradSqCoupled.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MATGRADSQCOUPLED_H
+#define MATGRADSQCOUPLED_H
```

Comment:
Please rename this to MatGradSquareCoupled. The `Sq` is an (IMO) unnecessary abbreviation.